tests/test_spotify.py

```diff
@@ -185,5 +185,51 @@ def test_that_song_function_returns_None_when_error(self):
         self.assertRaises(SpotifyNotRunning, x)
 
 
+class WebTests(unittest.TestCase):
+    """
+    Unit tests for Chrome (with extension) for spotify web player
+    """
+
+    def setup(self):
+        pass
+
+    @patch('SwSpotify.spotify.get_info_web')
+    def test_that_artist_function_calls_get_info(self, mock):
+        """
+        test that test artist function calls get_info_chrome function
+        """
+        x = artist()
+        self.assertTrue(mock.called)
+
+    @patch('SwSpotify.spotify.get_info_web')
+    def test_that_song_function_calls_get_info(self, mock):
+        """
+        test that test song function calls get_info_web function
+        """
+        x = song()
+        self.assertTrue(mock.called)
+
+    def test_that_artist_function_returns_None_when_error(self):
+        """
+        test that test artist function returns None when the get_info_web function will return an error
+        """
+        x = artist
+        self.assertRaises(SpotifyNotRunning, x)
```

Comment:
I don't think this test is relevant to the Web part since when the artist function is called, the exception would be raised by the native function and not the web function